libs/deal-ticket/src/type-selector.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button, FormGroup } from '@vegaprotocol/ui-toolkit';
+import { Order, OrderType } from './use-order-state';
+
+interface TypeSelectorProps {
+  order: Order;
+  onSelect: (type: OrderType) => void;
+}
+
+export const TypeSelector = ({ order, onSelect }: TypeSelectorProps) => {
+  return (
+    <FormGroup label="Order type">
+      <div className="flex gap-8">
```

Comment:
The component inside the FormGroup is identical to the one inside https://github.com/vegaprotocol/frontend-monorepo/pull/82/files#diff-44dcb60c83770bb6d57d8e259e005c644fc29177f0be35c135086f208aa6a94aR12-R32